src/helpers/tokenAuthentication.js

```diff
@@ -0,0 +1,37 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+
+class TokenAuthenticate {
+    static generateToken = (userDetails, expires) => {
+        const token = jwt.sign(userDetails,
+            process.env.SECRET, { expiresIn: expires }
+        );
+    return token;
+    }
+
+    static async tokenVerify (req, res, next) {
+        const token = req.headers['authorization']|| req.headers['Authorization'] || req.headers['x-access-token'] || req.query.token;
```

Comment:
It's very unlikely to send `Authorization` as `authorization`. You can just remove `req.headers['authorization']`. Instead you can check if the token is present in the body `req.body.token`

Reply:
Done